devicemodel/core/main.c

```diff
@@ -991,6 +989,9 @@ main(int argc, char *argv[])
 		case CMD_OPT_WINDOWS:
 			is_winvm = true;
 			break;
+		case CMD_OPT_FORCE_VIRTIO_MSI:
+			virtio_msix = 0;
```

Comment:
Wouldn't make sense to also update this variable to `virtio_msi`?

Out of curiosity, why we do provide a function (`virtio_uses_msix`) instead of using the variable directly? The function does not do anything else than return the variable binary value.

Reply:
Modify ‘--virtio_msix' beacuse it may mislead user that if it's set msix will be used in virtio when msi is accually in use.  Since users usually use msix, they don't have to pay attention to ’--vistio_msi', which is convenient for users. 
'virtio_msix' is kept because it accurately expresses to the developer that msix will be used.

For virtio_uses_msix, though I'm not the original author and don't know the excat reason ， I think it's reasonable.Firstly, we prefer to use static variables with controllable visibility. Secondly, the encapsulated function interface reserves a good space for adding judgment logic in the future.